Formula/pdnsrec.rb

```diff
@@ -12,40 +12,37 @@ class Pdnsrec < Formula
     sha256 "a9faf9edf0e71de5e9ec9fa70d27811f353a8451b41746208fbdc0d592aa5910" => :mountain_lion
   end
 
-  depends_on :macos => :lion
+  depends_on "pkg-config" => :build
   depends_on "boost"
-  depends_on "lua" => :optional
+  depends_on "openssl"
+  depends_on "lua"
+  depends_on "gcc" => :linked if MacOS.version <= :mavericks
```

Comment:
I know it does, but I do not consider that a sufficient reason for a hard dependency on gcc in the formula for Mavericks, unless it could be scoped to the bottle, which it can't be. Fails_with is supposed to provide flexibility not lock in a specific compiler dependency.